src/themes/su2c.ts

```diff
@@ -134,9 +138,18 @@ export const TYPOGRAPHY: TypographyType = {
     },
   ],
   fontFamilyHeadings: "itc_avant_garde_pro_mdbold,Arial,sans-serif",
+  fontFamilyLinks: "itc_avant_garde_pro_mdbold,Arial,sans-serif",
```

Comment:
The new font for SU2C looks considerably heavier. Why have we made this change?

Old:
![image](https://github.com/CRUKorg/cruk-react-components/assets/102517887/42c013b1-9db8-452a-ac93-e34496cbbd3b)


New:
![image](https://github.com/CRUKorg/cruk-react-components/assets/102517887/8f942fd3-4a1f-44c3-8e63-7f354f7a560b)